website/settings/defaults.py

```diff
@@ -1992,6 +1993,11 @@ class CeleryConfig:
 SPAM_THROTTLE_AUTOBAN = True
 SPAM_CREATION_THROTTLE_LIMIT = 5
 
+# CEDAR API configs
+CEDAR_API_HOST = ''
+CEDAR_API_KEY = ''
+CEDAR_HOME_FOLDER_ID = ''
```

Comment:
Is it documented somewhere (not GitHub, ofc) what these should be set to?

Reply:
hmm, let me check with @adlius , what's the best place to document this, I was thinking that the release playbook will have it.

A question just to double check: did you guys set this for staging? (I think so, otherwise it won't work for admin to pull stuff in)

Reply:
This was set for staging 3 and was communicated on a JIRA ticket.